packages/experiments/src/components/MicroFeedback/MicroFeedback.styles.ts

```diff
@@ -1,36 +1,62 @@
-import { IMicroFeedbackStyleProps, IMicroFeedbackStyles } from './MicroFeedback.types';
+import { IMicroFeedbackComponent, IMicroFeedbackStylesReturnType, IMicroFeedbackTokenReturnType } from './MicroFeedback.types';
 
-export const getStyles = (props: IMicroFeedbackStyleProps): IMicroFeedbackStyles => {
+const baseTokens: IMicroFeedbackComponent['tokens'] = {
+  questionMargin: '20px 12px 6px'
+};
+
+const inlineTokens: IMicroFeedbackComponent['tokens'] = {
+  followUpBackgroundColor: '#F3F2F1',
+  questionMargin: '12px 12px 6px',
+  width: '100%'
+};
+
+export const MicroFeedbackTokens: IMicroFeedbackComponent['tokens'] = (props, theme): IMicroFeedbackTokenReturnType => [
+  baseTokens,
+  props.inline && inlineTokens
+];
+
+export const MicroFeedbackStyles: IMicroFeedbackComponent['styles'] = (props, theme, tokens): IMicroFeedbackStylesReturnType => {
   return {
     root: {
       margin: '4px 8px 4px 8px',
+      width: tokens.width
+    },
+    iconContainer: {
       float: 'right'
     },
     followUpQuestion: {
       fontFamily: "'Segoe UI Semibold'",
-      margin: '20px 12px 6px 12px',
+      margin: tokens.questionMargin,
       fontSize: '12px',
       letterSpacing: '0.1px',
       border: '0px'
     },
-    followUpOptionText: {
-      fontFamily: "'Segoe UI'",
-      margin: '4px 0px 4px 0px',
-      fontSize: '12px',
-      letterSpacing: '0.11px',
-      textAlign: 'left',
-      border: '0px',
-      width: '100%'
+    followUpContainer: {
+      backgroundColor: tokens.followUpBackgroundColor,
+      paddingBottom: '6px'
+    },
+    thanksContainer: {
+      backgroundColor: 'transparent',
+      paddingTop: '6px',
+      paddingLeft: '6px',
+      paddingRight: '6px',
+      paddingBottom: '6px'
```

Comment:
If padding is `6px` across all directions you can just specify `padding: '6px'` instead of `paddingTop: '6px'`, `paddingBottom: '6px'`, etc.

Reply:
Fixed it and I wonder if this can be improved.